package.json

```diff
@@ -28,7 +29,8 @@
   },
   "scripts": {
     "serve-https-local": "serve-https -p 8080",
-    "pretest": "eslint src/ test/",
+    "lint": "eslint src/ test/",
+    "pretest": "npm run lint",
```

Comment:
I don't quite understand the reason for these changes? What benefit does the extra line of config give us?

Reply:
Perhaps this is partly a personal reflection, I'm used to most of the projects I've worked with having a lint script. I tend to run linting from the command line on a fairly regular basis. I had made the change when I was testing to make sure the config worked like it was supposed to.

I think it makes a bit more sense then runing `npm run pretest` from the command line to only run eslint.

I will remove it if you don't feel it's appropriate addition.

Reply:
Totally makes sense, and now I can see why it's useful. Thanks for taking the time to explain. Let's keep this.